symfony/framework-bundle/5.1/src/Kernel.php

```diff
@@ -17,12 +17,20 @@ protected function configureContainer(ContainerConfigurator $container): void
         $container->import('../config/{packages}/'.$this->environment.'/*.yaml');
         $container->import('../config/{services}.yaml');
         $container->import('../config/{services}_'.$this->environment.'.yaml');
+
+        if (file_exists($path = dirname(__DIR__).'/config/services.php')) {
```

Comment:
Since the recipes contains the file, it's always here.

Reply:
What about existing projects that will adopt the new recipes?

Reply:
The new file will come together

Reply:
> `$container->import('../config/{services}.yaml');`

this line is using `{...}` *because* we want it to load `services.yaml` conditionally. I think we want the same there: not forcing ppl to update their kernel if they don't want to use `services.php`.

> The new file will come together

Oh, and this new file will break existing projects, which will have a wtf moment because of it breaking their config. How do we prevent that?

Reply:
updated